code/__DEFINES/dcs/signals/signals_object.dm

```diff
@@ -503,3 +503,6 @@
 
 /// from /datum/component/dart_insert/on_reskin()
 #define COMSIG_DART_INSERT_PARENT_RESKINNED "dart_insert_parent_reskinned"
+
+/// Fired when the user wants to exit their current object possession. From /proc/release().
+#define COMSIG_END_OBJECT_POSSESSION "end_object_possession"
```

Comment:
i think the main issue with this signal was that it was the same thing being sent from multiple locations, i moved it to be a different signal based on the origination in case we want to do something different in the future.